providers/apache/kafka/src/airflow/providers/apache/kafka/queues/kafka.py

```diff
@@ -35,44 +35,35 @@ class KafkaMessageQueueProvider(BaseMessageQueueProvider):
     """
     Configuration for Apache Kafka integration with common-messaging.
 
-    It uses the ``kafka://`` URI scheme for identifying Kafka queues.
+    [START kafka_message_queue_provider_description]
 
-    **URI Format**:
-
-    .. code-block:: text
-
-        kafka://<broker>/<topic_list>
-
-    Where:
-
-    * ``broker``: Kafka brokers (hostname:port)
-    * ``topic_list``: Comma-separated list of Kafka topics to consume messages from
-
-    **Examples**:
-
-    .. code-block:: text
-
-        kafka://localhost:9092/my_topic
-
-    **Required kwargs**:
-
-    * ``apply_function``: Function to process each Kafka message
-
-    You can also provide ``topics`` directly in kwargs instead of in the URI.
+    * It uses the ``kafka`` as scheme for identifying Kafka queues.
+    * For parameter definitions take a look at :class:`~airflow.providers.apache.kafka.triggers.await_message.AwaitMessageTrigger`.
 
     .. code-block:: python
 
         from airflow.providers.common.messaging.triggers.msg_queue import MessageQueueTrigger
+        from airflow.sdk import Asset, AssetWatcher
 
+        # New scheme-based approach
```

Comment:
I do not think it is worth mentioning it

```suggestion
        # New scheme-based approach
```